nucypher/characters/lawful.py

```diff
@@ -1470,20 +1484,29 @@ def encrypt_for_pre(
         return message_kit
 
     def encrypt_for_dkg(
-        self, plaintext: bytes, conditions: Lingo
+        self,
+        plaintext: bytes,
+        conditions: Lingo,
     ) -> ThresholdMessageKit:
+        if not self.signer:
+            raise TypeError("This Enrico doesn't have a signer.")
```

Comment:
There's missing commit authorship in this line 🤖

Reply:
[wdym](https://github.com/KPrasch/nucypher/commit/783f2dcd76fec7fe1bf78ab032f2ac10b879a3f5)?